.github/workflows/e2e.yml

```diff
@@ -0,0 +1,59 @@
+name: e2e tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    # cypress requires xvfb
+    container: thompsnm/nodejs-chrome-xvfb
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: XXXX
```

Comment:
i might make the user and db explicit (POSTGRES_USER, POSTGRES_DB)

Reply:
ok, fixed here https://github.com/salsita/foosball-rating/pull/154/commits/31a8ba5da60da761aea1132031a4890dfdba22cd